src/Curiosity/Runtime.hs

```diff
@@ -268,7 +273,14 @@ handleCommand runtime@Runtime {..} display user command = do
               let value' = if useHs
                     then show value
                     else LT.toStrict (Aeson.encodeToLazyText value)
-              display value'
+              if short
+                then
+                  display
+                  $  User.unUserId (User._userProfileId value)
+                  <> " "
+                  <> User.unUserName
+                       (User._userCredsName $ User._userProfileCreds value)
```

Comment:
Maybe this can be broken out into a `where` to keep this more neat?

Reply:
This will change in the branch introducing golden testing. But I think you're right and should be part of the goal of making the code really nice.